core/templates/pages/splash-page/splash-page.component.spec.ts

```diff
@@ -96,8 +96,6 @@ describe('Splash Page', () => {
     }).compileComponents();
   });
 
-  beforeEach(angular.mock.module('oppia'));
```

Comment:
Is this change expected?

Reply:
We don't need that line in migrated spec files so I removed it